src/common/actions/snaps.js

```diff
@@ -9,16 +9,15 @@ export const FETCH_SNAPS = 'FETCH_REPOSITORIES';
 export const FETCH_SNAPS_SUCCESS = 'FETCH_SNAPS_SUCCESS';
 export const FETCH_SNAPS_ERROR = 'FETCH_SNAPS_ERROR';
 
-export function fetchUserSnaps(owner) {
+export function fetchUserSnaps() {
```

Comment:
As I said on IRC yesterday, I think this is a mistake.  While currently it's true that the owner here is always identical to the logged-in user, this will need to stop being the case as soon as we implement organisation support.